src/neuroconv/nwbconverter.py

```diff
@@ -160,14 +178,43 @@ def run_conversion(
             nwbfile=nwbfile,
             metadata=metadata,
             overwrite=overwrite,
-            verbose=self.verbose,
+            backend=backend,
+            verbose=getattr(self, "verbose", False),
```

Comment:
?

Reply:
It fixed a test failure somewhere 🤷

There were a lot of those kind of things here

Reply:
Interesting.